### PR TITLE
feat: add option to open chatbot directly by using a link

### PR DESCRIPTION
**Pull request by radireddy**
option to open chatbot directly using a link

ref: MANAGER-8219

Signed-off-by: Ravindra Adireddy <ravindra.adireddy@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8219
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/account-sidebar/src/controller.js [hunk 1]
```diff
@@ -26,11 +26,17 @@ export default class OvhManagerAccountSidebarCtrl {
       this.sidebarExpand = false;
     });
 
-    const unregisterListener = this.$rootScope.$on('ovh-chatbot:enable', () => {
-      this.hasChatbot = true;
-      this.links = this.getLinks();
-      unregisterListener();
-    });
+    const unregisterListener = this.$rootScope.$on(
```
Comment (frenautvh) [inline packages/manager/modules/account-sidebar/src/controller.js:29]: it looks like you are adding arguments to the ovh-chatbot:enable event but i don't see that you are updating and passing new arguments where the events are fired, is it normal ?

Comment (cbourgois) [reply packages/manager/modules/account-sidebar/src/controller.js:29]: Here : https://github.com/ovh/manager/pull/6498/files#diff-4ca5b53f5c5ac2a716e2fc229ab16cb90edc260bd08463e2c01120839825d72eR46-R50

File: packages/manager/modules/account-sidebar/src/controller.js [hunk 2]
```diff
@@ -26,11 +26,17 @@ export default class OvhManagerAccountSidebarCtrl {
...
+      'ovh-chatbot:enable',
+      (event, hasChatbot, showLivechat) => {
+        this.hasChatbot = true;
+        this.links = this.getLinks();
+        unregisterListener();
+        if (this.hasChatbot && showLivechat) {
```
Comment (cbourgois) [inline packages/manager/modules/account-sidebar/src/controller.js:35]: As we update `this.hasChatbot` with `true` 3 lines before, maybe we can remove the `this.hasChatbot` from the condition ?

**Comment by cbourgois** [comment]
I'm a bit confused about `showLivechat` resolvable and using the `livechatService`. Maybe we can use only the resolvable, get the value in controller and spread the information through the event ? Or maybe I miss something :)

**Review by cbourgois — DISMISSED** [review]

File: packages/manager/modules/account-sidebar/src/controller.js [hunk 3]
```diff
@@ -49,6 +56,7 @@ export default class OvhManagerAccountSidebarCtrl {
   }
 
   openChatbot() {
+    console.log('opening chatbot');
```
Comment (marie-j) [inline packages/manager/modules/account-sidebar/src/controller.js:59]:
```suggestion

```

File: packages/manager/modules/account-sidebar/src/controller.js [hunk 4]
```diff
@@ -26,11 +26,18 @@ export default class OvhManagerAccountSidebarCtrl {
...
-      this.links = this.getLinks();
-      unregisterListener();
-    });
+    const unregisterListener = this.$rootScope.$on(
+      'ovh-chatbot:enable',
+      (event, hasChatbot, showLivechat) => {
+        this.hasChatbot = true;
+        this.links = this.getLinks();
+        unregisterListener();
+        console.log('showLivechat', showLivechat);
```
Comment (marie-j) [inline packages/manager/modules/account-sidebar/src/controller.js:35]:
```suggestion

```

**Review by marie-j — CHANGES_REQUESTED** [review]

**Review by cbourgois — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**